docs/mega-linter-runner.md

```diff
@@ -44,7 +45,7 @@ You can run mega-linter-runner without installation by using `npx`
 Example:
 
 ```shell
-npx mega-linter-runner -r beta -e 'ENABLE=MARKDOWN,YAML' -e 'SHOW_ELAPSED_TIME=true'
+npx mega-linter-runner -r beta -e "'ENABLE=MARKDOWN,YAML'" -e 'SHOW_ELAPSED_TIME=true'
```

Comment:
This files is generated from megalinter-runner/README.md , please can you update it ? :)

Reply:
@nvuillam done :)